.github/workflows/x-create-tags.yml

```
@@ -0,0 +1,57 @@
+name: X Create tags
+on:
+  workflow_call:
+    inputs:
+      tag:
+        required: true
+        type: string
+      branch:
+        required: false
+        type: string
+        default: main
+      create-commit:
```

I'd rename this to make it more explicit, for example:

```suggestion
      set-versions:
```